src/components/GoToSpringboardModal.vue

```diff
@@ -32,34 +32,32 @@
 </template>
 
 <script setup>
-import { ref } from 'vue'
-import { useModals } from '../composables/useModals'
-import { EButtonType, EModal } from '../types/ui'
+import { EButtonType } from '../types/ui'
 import BaseButton from './BaseButton.vue'
 import BaseModal from './BaseModal.vue'
-
-const { visibleModal, hideModal } = useModals()
+import { springBoardUrl } from '../utils/springboardConfiguration'
 
 const props = defineProps({
   appName: {
     type: String,
     required: true
   },
-
-  springboardUrl: {
+  environment: {
     type: String,
     required: true
-  }
+  }  
 })
 
 const emit = defineEmits(['login'])
 
 function handleSpringboardLogin() {
-  hideModal()
+  emit('login')
```

Comment:
Just want to double check that emitting 'login' here hides this modal from the parent component? Why change from the pattern of using the `useModals` hook and `EModal.go_to_springboard`

Reply:
It's an issue of competing patterns. We are using the uiStore and Modals component in most places in cloud console and holofuel. I like the composable pattern better but this would be the only place where we use it.